composer.json

```diff
@@ -13,8 +13,8 @@
         }
     },
     "replace": {
-        "guzzlehttp/guzzle": "*",
-        "guzzlehttp/psr7": "*"
+        "guzzlehttp/guzzle": "^7.5",
```

Comment:
Note: it might be nice to be able to specify "^5.3|^7.5" because core 10.10.0 has guzzle 5.3. But composer complains about that because `roave/security-advisories` nowadays refuses to allow guzzle v5 at all.

When we next release `files_external_s3` we need to either require core min version 10.11, or carefully check if it all still works with core 10.10.